lib/rhc/commands/app.rb

```diff
@@ -425,7 +425,8 @@ def show(app_name)
         case options.gears
         when 'quota'
           opts = {:as => :gear, :split_cells_on => /\s*\t/, :header => ['Gear', 'Cartridges', 'Used', 'Limit'], :align => [nil, nil, :right, :right]}
-          table_from_gears('echo "$(du --block-size=1 -s 2>/dev/null | cut -f 1)"', groups, opts) do |gear, data, group|
+          # quota 'blocks' are actually kb, so have to multiply by 1024, see 'man quota'
+          table_from_gears('echo "$(quota | egrep -v \'quotas|Filesystem\' | awk \'{ print $2 * 1024 }\')"', groups, opts) do |gear, data, group|
```

Comment:
Oh, you need to use `echo "$(quota ...)"` because `oo-trap-user` recognises the `quota` command and runs it with `execv` (cf. https://github.com/openshift/origin-server/blob/master/node/misc/bin/oo-trap-user#L310-L335) whereas `oo-trap-user` does not recognise `echo` and instead runs it with `bash -c`, so pipes work (cf. https://github.com/openshift/origin-server/blob/master/node/misc/bin/oo-trap-user#L216-L224).

`command quota | awk 'NR == 3 { print $2 * 1024 }'` would also work, but since we're using `echo "$(foo)"` elsewhere we can stick to that idiom.